jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -786,6 +786,65 @@ proxy:
     - "<vpc-or-subnet-cidr>"   # VPC or subnets to exclude from the proxy (optional)
 ----
 
+=== n. Encrypting Environment Variables
+
+All environment variables uses in a job are encrypted using either Hashicorp https://www.vaultproject.io/[Vault] or Google https://developers.google.com/tink[Tink]. By Default, CircleCI Server 4.x will use Vault to generate and store encryption keys.
+
+==== Using Tink
+
+If you wish to use Tink instead of Vault you will need to first enable it in your `values.yaml`. 
+
+[source,yaml]
+----
+tink:
+  enabled: false
+  keyset: ""
+----
+
+Tink is given precedence over vault. If `tink.enabled` is true, Vault will not be deployed.
+
+WARNING: Tink or Vault must be set once at install and cannot be changed after deployment.
```

Comment:
I want to include something here about how important the key is once generated and how contexts will need to be completely rolled if lost